TerrorConsole/Assets/_Main/Scripts/Dialogue/DialogueTrigger.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class DialogueTrigger : MonoBehaviour
+    {
+        public DialogueData dialogueData;
+
+        public void TriggerDialogue()
+        {
+            FindObjectOfType<DialogueManager>().StartDialogue(dialogueData);
```

Comment:
We need to change this to use our dependency solution, we can connect it together.